shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/providers/CustomerNamespaceProviderImplTest.java

```diff
@@ -92,9 +95,18 @@ void testNamespaceDeletedWhenEmpty() {
         final BridgeDTO dto = TestSupport.newRequestedBridgeDTO();
         dto.setCustomerId("hofstadter");
         bridgeIngressService.createBridgeIngress(dto);
+        // Wait until BridgeIngress is really created, can take some time due to client caching - https://issues.redhat.com/browse/MGDOBR-308
+        Awaitility.await()
+                .atMost(Duration.ofSeconds(5))
```

Comment:
is 5 sec enough? We can enlarge it a bit just to be 100% sure, but up to you

Reply:
It should be enough, as `BridgeIngress` should be created by `bridgeIngressService.createBridgeIngress(dto)`. The resource should be available in <=100ms